_sources/functions_in_cpp/mixed_up_code.rst

```diff
@@ -0,0 +1,344 @@
+Mixed Up Code Exercises
+-----------------------
+
+Answer the following **Mixed-Up Code** questions to practice what you have learned in this chapter.
+
+.. parsonsprob:: functions_in_cpp_mixed_up_code_1
+   :numbered: left
+   :adaptive:
+   :noindent:
+   :practice: T
+
+   Vacation time! But before you go, you need to convert your currency.
+   Let's write the code for the dollarToYen function. dollarToYen
+   takes dollar as a parameter and returns the equivalent amount of Japanese yen.
+   The conversion rate is 1 USD equals 105.42 Japanese yen.
+   Put the necessary blocks of code in the correct order.
```

Comment:
Update the formatting of these instructions so that the name of the functions & variables are in code format (e.g., ``dollarToYen`` function)